app/models/manageiq/providers/ansible_tower/automation_manager/configuration_script_source.rb

```diff
@@ -0,0 +1,3 @@
+class ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScriptSource < ConfigurationScriptSource
+  has_many :playbooks
```

Comment:
yea, right and `ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScriptSource` does not work because there is no STI type column yet. So I'm gonna remove this.